minimesos/src/main/java/com/containersol/minimesos/mesos/MesosMaster.java

```diff
@@ -48,6 +48,7 @@ public int getPortNumber() {
         TreeMap<String,String> envs = new TreeMap<>();
         envs.put("MESOS_QUORUM", "1");
         envs.put("MESOS_ZK", getFormattedZKAddress());
+        envs.put("MESOS_LOGGING_LEVEL", config.getLoggingLevel());
```

Comment:
Why do we play with environment variables in both `MesosMaster` and `MesosContainer`?